api/BUILD

```diff
@@ -133,12 +133,18 @@ proto_library(
         "//envoy/config/common/matcher/v3:pkg",
         "//envoy/config/core/v3:pkg",
         "//envoy/config/endpoint/v3:pkg",
+        "//envoy/config/filter/thrift/router/v2alpha1:pkg",
         "//envoy/config/grpc_credential/v3:pkg",
+        "//envoy/config/health_checker/redis/v2:pkg",
         "//envoy/config/listener/v3:pkg",
         "//envoy/config/metrics/v3:pkg",
         "//envoy/config/overload/v3:pkg",
         "//envoy/config/ratelimit/v3:pkg",
         "//envoy/config/rbac/v3:pkg",
+        "//envoy/config/resource_monitor/fixed_heap/v2alpha:pkg",
+        "//envoy/config/resource_monitor/injected_resource/v2alpha:pkg",
+        "//envoy/config/retry/omit_canary_hosts/v2:pkg",
+        "//envoy/config/retry/previous_hosts/v2:pkg",
```

Comment:
I guess these changes are not relevant to this PR and explain the doc generation failures.

Reply:
Thanks @rojkov - I've verified that the api/BUILD change was the culprit.  It was triggered (but not committed) by the git-push automation, `./tools/proto_format/proto_format.sh fix` also causes it.   It's reverted for now in my feature branch, to get CI passing for this PR.